apps/gruenbuch/models.py

```diff
@@ -18,17 +18,27 @@ class GruenbuchIndexPage(Page):
     def book(self):
         return self
 
+    @property
+    def isEmpty(self):
+        if self.get_children():
+            return False
+        else:
+            return True
+
     @property
     def chapters(self):
-        return self.get_children().specific()
+        if self.get_children():
+            return self.get_children().specific()
 
     @property
     def active_chapter(self):
-        return self.chapters.first()
+        if self.chapters:
```

Comment:
I find it somehow weird that the properties always ask for the one before. On the other hand, they also use what they ask for, so, keep that!
But I think, they should always return something either in an else or just by indentation.
```
    @property
    def active_chapter(self):
        if self.chapters:
            return self.chapters.first()
        return ''
```
I am not sure, what a good empty value to return would be, though. The empty string could work.